src/wp-content/themes/twentyfifteen/css/blocks.css

```diff
@@ -335,6 +340,11 @@ p.has-drop-cap:not(:focus)::first-letter {
 	text-transform: none;
 }
 
+.wp-block-pullquote[class*="-text-color"] blockquote cite,
```

Comment:
Let's discuss possible changes to the Pullquote styles separately, on [59801](https://core.trac.wordpress.org/ticket/59801).

Reply:
Sorry I did not check about this ticket. 

Now I have Reverted Pullquote block's all changes